frontend/islands/vite.config.mts

```diff
@@ -31,11 +29,13 @@ const ReactCompilerConfig = {
 // https://vitejs.dev/config/
 export default defineConfig({
   resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "../src"),
+    },
```

Comment:
`rolldown-vite` natively supports resolution using the `paths` from tsconfig:

  ```ts
resolve: {
    tsconfigPaths: true,
  },
  experimental: {
    enableNativePlugin: true,
  },
```

This requires the `enableNativePlugin=true` currently.